internal/pkg/filesystem/aferofs/utils_test.go

```diff
@@ -9,46 +9,152 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/log"
 )
 
-func TestCopyFs2FsRootToRoot(t *testing.T) {
+func TestCopyFs2FsRootToRoot_LocalToMemory(t *testing.T) {
 	t.Parallel()
 	tempDir := t.TempDir()
-	localFs, err := NewLocalFs(log.NewNopLogger(), tempDir, `/`)
+	localFs, err := NewLocalFs(log.NewNopLogger(), tempDir, "/")
 	assert.NoError(t, err)
-	memoryFs, err := NewMemoryFs(log.NewNopLogger(), `/`)
+	memoryFs, err := NewMemoryFs(log.NewNopLogger(), "/")
 	assert.NoError(t, err)
 
 	// Create files
-	assert.NoError(t, localFs.WriteFile(filesystem.NewRawFile(`foo.txt`, `content1`)))
-	assert.NoError(t, localFs.WriteFile(filesystem.NewRawFile(filesystem.Join(`my-dir`, `bar.txt`), `content2`)))
+	assert.NoError(t, localFs.WriteFile(filesystem.NewRawFile("foo.txt", "content1")))
+	assert.NoError(t, localFs.WriteFile(filesystem.NewRawFile(filesystem.Join("my-dir", "bar.txt"), "content2")))
 
 	// Copy
-	assert.NoError(t, CopyFs2Fs(localFs, ``, memoryFs, ``))
+	assert.NoError(t, CopyFs2Fs(localFs, "", memoryFs, ""))
 
 	// Assert
-	file1, err := memoryFs.ReadFile(filesystem.NewFileDef(`foo.txt`))
+	file1, err := memoryFs.ReadFile(filesystem.NewFileDef("foo.txt"))
 	assert.NoError(t, err)
-	assert.Equal(t, `content1`, file1.Content)
-	file2, err := memoryFs.ReadFile(filesystem.NewFileDef(filesystem.Join(`my-dir`, `bar.txt`)))
+	assert.Equal(t, "content1", file1.Content)
+	file2, err := memoryFs.ReadFile(filesystem.NewFileDef(filesystem.Join("my-dir", "bar.txt")))
 	assert.NoError(t, err)
-	assert.Equal(t, `content2`, file2.Content)
+	assert.Equal(t, "content2", file2.Content)
+}
+
+func TestCopyFs2FsRootToRoot_LocalToLocal(t *testing.T) {
+	t.Parallel()
+	tempDir1 := t.TempDir()
+	localFs1, err := NewLocalFs(log.NewNopLogger(), tempDir1, "/")
+	assert.NoError(t, err)
+	tempDir2 := t.TempDir()
+	localFs2, err := NewLocalFs(log.NewNopLogger(), tempDir2, "/")
+	assert.NoError(t, err)
+
+	// Create files
+	assert.NoError(t, localFs1.WriteFile(filesystem.NewRawFile("foo.txt", "content1")))
+	assert.NoError(t, localFs1.WriteFile(filesystem.NewRawFile(filesystem.Join("my-dir", "bar.txt"), "content2")))
+
+	// Copy
+	assert.NoError(t, CopyFs2Fs(localFs1, "", localFs2, ""))
+
+	// Assert
+	file1, err := localFs2.ReadFile(filesystem.NewFileDef("foo.txt"))
+	assert.NoError(t, err)
+	assert.Equal(t, "content1", file1.Content)
+	file2, err := localFs2.ReadFile(filesystem.NewFileDef(filesystem.Join("my-dir", "bar.txt")))
+	assert.NoError(t, err)
+	assert.Equal(t, "content2", file2.Content)
+}
+
+func TestCopyFs2FsRootToRoot_BaseToBase(t *testing.T) {
```

Comment:
Podarilo sa mi to replikovat v tomto test.
Bug je v `BaseFs` (`SubDirFs`), mozno priamo v `afero` libke, skumam dalej ...

**So samotnym kopirovanim nie je problem, ale v `chmod` sa tam duplikuje `basePath`**:
![image](https://user-images.githubusercontent.com/19371734/188607455-b28889e6-322e-4b01-81e2-8399feb6e2dd.png)

Reply:
oomg, díky! Vezmu ti za to aspoň nějaký ticket z L2. 🙂